app/policies/organizations/external_form_upload_policy.rb

```diff
@@ -0,0 +1,7 @@
+module Organizations
+  class ExternalFormUploadPolicy < ApplicationPolicy
```

Comment:
Can you add the pre-checks here for organization and staff, please - see the Dashboard policy for example.